deploy/docker-compose.yaml

```diff
@@ -0,0 +1,76 @@
+version: '3.7'
+
+# This docker-compose file brings up a full Observation Portal stack with Task Scheduler,
+# Task Worker, and Example Frontend. There is also an associated container of redis for
+# caching and task scheduling. You will want to modify the DB_* environment variables to
+# connect to your db backend, and modify the *_URL environment variables to point to the
+# location where you have the associated services running, like CONFIGDB.
+services:
+    redis:
+        image: redis:3.2
+        command: ["redis-server", "--appendonly", "yes"]
+        restart: always
+        healthcheck:
+            test: ["CMD", "redis-cli", "ping"]
+            interval: 10s
+            timeout: 5s
+            retries: 30
+        expose:
+            -  "6379"
+        volumes:
+            -  ./data/redis:/data
+
+    nginx:
+        image: nginx:1.19.0-alpine
+        ports:
+            - 8000:80
+        volumes:
+            - ./nginx.conf:/etc/nginx/conf.d/default.conf
+            - static_volume:/static
+        depends_on:
+            - observation_portal
+
+    observation_portal:
+        image: observatorycontrolsystem/observation-portal:3.5.4
```

Comment:
I think it would be a good idea to also allow the user to configure the image repo and image tag from a single location, maybe from a separate env file or something

Reply:
Yeah should work